start.sh

```diff
@@ -1,9 +1,5 @@
 #!/bin/bash
 
-
-LISTEN_PORT=${LISTEN_PORT:-8181}
```

Comment:
snapteld will use the SNAP_PORT environment variable if it is set, so we dont need to modify the config.

see `snapteld -h` for details